frontend/public/components/image-stream-tag.tsx

```diff
@@ -141,6 +148,39 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
           </div>
         )}
       </div>
+      <div className="co-m-pane__body-group">
+        <SectionHeading text={t('public~Supported Platforms')} />
```

Comment:
So this approach wont enable column based sorting of the table.
Please check how other tables are implemented, for example how [pod details page](https://github.com/openshift/console/blob/master/frontend/public/components/pod.tsx#L857) is implementing [VolumesTable](https://github.com/openshift/console/blob/master/frontend/public/components/volumes-table.tsx#L167)

Reply:
Thanks, I presorted the columns - I can add per citation. thank you

Reply:
I've updated based on VolumesTable's style/code